ush/land/letkf_create_ens.py

```diff
@@ -29,7 +29,7 @@
 for (mem, value) in zip(ens_dirs, sign):
     for tt in range(1, 7):
         # open file
-        out_netcdf = os.path.join(workdir, mem, f"{fstub}.sfc_data.tile{tt}.nc")
+        out_netcdf = os.path.join(workdir, mem, "RESTART", f"{fstub}.sfc_data.tile{tt}.nc")
```

Comment:
I include this script in `land_analysis.py` in the global-workflow rather than this floating script.

Reply:
Do you mean you have put the script `letkf_create_ens.py` in `land_analysis.py`? The script `letkf_create_ens.py` comes from GDASApp, and this script is a key part in GDASApp for the snow DA tests.

Reply:
Yes.  See PR #1635 in global-workflow

Reply:
@CoryMartin-NOAA Do you think it is okay we can keep this letkf_create_ens.py both separately in the GDASApp and in the land_analysis.py from global-workflow?

Reply:
We should not duplicate efforts. The GDASApp has several tests that use global-workflow scripts, we can make the GDASApp land test that uses this .py file do that instead (once it is in the workflow develop branch)

Reply:
@CoryMartin-NOAA I need your help to make the GDASApp land test by using the global-workflow script. Thanks.

Reply:
@aerorahul This PR is the companion with the https://github.com/NOAA-EMC/global-workflow/pull/1636. When I use your implementation in your draft PR, this script will be removed.